filetests/isa/x86/vconst.clif

```diff
@@ -0,0 +1,11 @@
+test binemit
+set opt_level=best
+set enable_simd
+target x86_64
+
+function %test_vconst_b8() {
+ebb0:
+[-, %xmm2]  v0 = vconst.b8x16 0x00  ; bin: 0f 10 15 fffffff9 PCRelRodata4(0)
+[-, %xmm3]  v1 = vconst.b8x16 0x01  ; bin: 0f 10 1d 00000002 PCRelRodata4(16)
+            return
```

Comment:
Can we add a way to check the readonly data emitted?

Reply:
How, though? I can't figure out how to get at the binemitted rodata after the function.

Reply:
I mean adding a new kind of directive to [filetest](https://github.com/CraneStation/cranelift/tree/master/cranelift-filetests/src).

Reply:
Let me know what you think of af2edc9--especially the formatting. Seems like comparing long vectors could get unwieldy.